### DISCUSSION TITLE
Is there any way to adapt each items size to the content height?

### DISCUSSION THREAD

**Discussion by bkerz** [react-grid-layout/react-grid-layout#1717] [General]
Hello, I'm using this library but I wanted that the items adapt to the content height to avoid scrolling when there is much content. Do you know if there's currently a way to do it?

For example, I have a text editor here but the content is hidden because of the fixed size the item has. But I would like to write more to make the container grow vertically

When I use `isResizable` prop i can do it manually, but i would like to do it automatically

![image](https://user-images.githubusercontent.com/48168047/166293465-91d6ea03-6e48-460d-bb63-5210694ad7bb.png)

**Comment by Henry129999** [comment]
you can use chrome ‘ResizeObserver’ api. Dynamic monitor the true height of the component. And set the true height to layouts. Final re-render React-Grid-Layout.
